core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1471,17 +1471,27 @@ class WorkspaceService(protected val userInfo: UserInfo,
     }
 
   def getSubmissionStatus(workspaceName: WorkspaceName, submissionId: String) = {
```

Comment:
I would like a return type annotation too.  Guessing that's a `Future[SubmissionStatusResponse]` ?

Reply:
Will add!

Reply:
IntelliJ says it returns a Future[PerRequestMessage]...

Reply:
ok